### PR TITLE
fix label rotation

### PR DESCRIPTION
**Pull request by pHill5136**

### CODE REVIEW

**Comment by pHill5136** [comment]
@csmoore please review and merge

File: source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/TabBaseViewModel.cs [hunk 1]
```diff
@@ -1071,6 +1071,51 @@ internal void AddTextToMap(IGeometry geom, string text)
             RaisePropertyChanged(() => HasMapGraphics);
         }
 
+        internal void AddTextToMap(IGeometry geom, string text, double angle, AzimuthTypes azimuthType)
+        {
+            var mxDoc = ArcMap.Application.Document as IMxDocument;
+            var av = mxDoc.FocusMap as IActiveView;
+            var gc = av as IGraphicsContainer;
+            double bearing = 0.0;
+            if(azimuthType == AzimuthTypes.Mils)
```
Comment (csmoore) [inline source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/TabBaseViewModel.cs]: Is there a method that already does this calculation/conversion somewhere that could be used instead?

File: source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/TabBaseViewModel.cs [hunk 2]
```diff
@@ -1071,6 +1071,51 @@ internal void AddTextToMap(IGeometry geom, string text)
...
+            {
+                bearing = angle * 0.05625;
+            }
+            else
+            {
+                bearing = angle;
+            }
+            double rotate = 360 - (bearing + 270.0) % 360;
```
Comment (csmoore) [inline source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/TabBaseViewModel.cs]: minor: could be just (450-bearing) % 360 - also maybe a comment about converting from bearing to XY angle

File: source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/TabBaseViewModel.cs (hunk 1 above)
Comment (pHill5136) [inline source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/TabBaseViewModel.cs]: I think all of the existing methods also convert from radians so I just put the formulas in this piece of code.

**Pull request merged**